### ISSUE TITLE
FDS Sprinkler - dissapearance of smoke and fire after sprinkler activation

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#806]
```
Please complete the following lines...

Application Version:FDS 5.3.0 Serial
SVN Revision Number:3193
Compile Date:Fri, 30 Jan 2009
Operating System:xp

Describe details of the issue below:

I use PYROSIM as graphical user interface

My problem is the following. I have a rectangular room (7.32m X 3.66m X  
2.44m), where a burner is set up near the left wall in the center plane.
Obstructions are set up around the burner in order to avoid to put out the 
fire with the sprinkler activation.

I have a sprinkler set up at the middle of the roof.

My problem is that, few seconds after the sprinkler activation, everything 
except the sprinkler spray dissapear instantaneously. I mean the smoke the 
fire (HRR go down to 0 instantaneously).

I try to sort out the problem and to change a lot of parameters but 
nothing change. Does anyone have an idea about my problem and can help me? 

Thank you very much 

I let the codes and 2 pictures to illustrate. Then sprinler went off at 35 
s. 
```

Original issue reported on code.google.com by `maticu@hotmail.fr` on 2009-07-24 14:47:33

<hr>
- _Attachment: [At 45.9s, so 10 seconds after SP act.bmp](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-831/comment-0/At 45.9s, so 10 seconds after SP act.bmp)_
- _Attachment: [At 46.3 s.bmp](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-831/comment-0/At 46.3 s.bmp)_
- _Attachment: [Thesis test 14 meshe 0,08m.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-831/comment-0/Thesis test 14 meshe 0,08m.fds)_

**Comment by gforney** [comment]
```
Run your case with the latest version of FDS and see if your problem still exists.
```

Original issue reported on code.google.com by `drjfloyd` on 2009-07-24 15:43:42

**Comment by gforney** [comment]
```
i download the latest version so i'll try, but it's really strange, i mean the spray

of the sprinkler change, it's normal and at one time it grows and it's precisely at

that time that everything stop.It's really strange
Moreover i don't understand really well the meaning of the two latitude angles when

i set up my simulation, but it doesn't mean that the spray changes its angle during

the simulation, isnt it? 

Thank you very much for your answer.
```

Original issue reported on code.google.com by `maticu@hotmail.fr` on 2009-07-24 19:36:21

**Comment by gforney** [comment]
```
Your sample input file doesn't have a sprinkler in it, so I am going to assume your
question is about SPRAY_ANGLE on PROP?  This is just the range of angles over which
water spray is introduced with 0 being the south pole of the sprinkler.  So
SPRAY_ANGLE=89,91  would be a sprinkler that discharged water in a thin horizontal
sheet.

The behavior you have observed is not all that strange from a numerics point of view.
 If something goes amiss in the calculation due to a bug, you can wind up with having
some number divided by zero or some other similar operation.  This type of error
tends to quickly propagate through other variables and tends to result in behavior
like all species going to zero in a single timestep.  As there have been some bug
fixes pertaining to water spray between 5.3 and 5.3.1, hopefully your bug has been
fixed already.
```

Original issue reported on code.google.com by `drjfloyd` on 2009-07-24 19:48:15

**Comment by gforney** [comment]
```
With no further updates as to whether or problem exists with latest version of FDS,
issue is being closed.
```

Original issue reported on code.google.com by `drjfloyd` on 2009-09-25 18:59:39
